addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1161,7 +1161,7 @@ def _create_sub_channel(self, from_message_id=None, name=None):
                 "channel_member_ids": [Command.create({"partner_id": self.env.user.partner_id.id})],
                 "channel_type": "channel",
                 "from_message_id": message.id,
-                "name": name or (message.body.striptags()[:30] if message else _("New Thread")),
+                "name": name or (message.body.striptags()[:30] if message and message.body else _("New Thread")),
```

Comment:
Empty recordsets can be chained with field access.
```suggestion
                "name": name or (message.body.striptags()[:30] if message.body else _("New Thread")),
```